src/lib/viewers/BaseViewer.js

```diff
@@ -1081,7 +1085,11 @@ class BaseViewer extends EventEmitter {
      */
     handleAnnotationControlsClick({ mode }) {
         const nextMode = this.annotationControlsFSM.transition(AnnotationInput.CLICK, mode);
-        this.annotator.toggleAnnotationMode(nextMode);
+        this.annotator.toggleAnnotationMode(
```

Comment:
Would it make more sense to somehow have this discoverability FF live in the FSM?

Reply:
I first want to put it in FSM too. But later I find we cannot do that because of the line below. The annotations control still needs to set mode to `none` instead of `region`.